internal/services/loadbalancer/loadbalancer_resource.go

```diff
@@ -191,10 +212,15 @@ func loadBalancerCreateUpdate(d *pluginsdk.ResourceData, meta interface{}) error
 		properties.FrontendIPConfigurations = frontendIPConfigurations
 	}
 
+	sku := network.LoadBalancerSku{
+		Name: network.LoadBalancerSkuName(d.Get("sku").(string)),
+	}
+
 	loadBalancer := network.LoadBalancer{
 		Name:                         pointer.FromString(id.Name),
 		Location:                     pointer.FromString(location.Normalize(d.Get("location").(string))),
 		Tags:                         tags.Expand(d.Get("tags").(map[string]interface{})),
+		Sku:                          &sku,
```

Comment:
could we set this inline? theres nor eason to have a variable for it

Reply:
Sure, will do that, thank you